packages/extension-code-block/src/code-block.ts

```diff
@@ -138,6 +138,12 @@ export const CodeBlock = Node.create<CodeBlockOptions>({
         }
 
         if (isAtStart || !$anchor.parent.textContent.length) {
+          if (this.editor.isActive('orderedList'))
```

Comment:
I think using defined extensions here to solve this is the wrong way because this issue will appear in other types of nodes too I guess.

Reply:
Oh, you are right. It's because I haven't thought about it comprehensively enough.

I'm currently modifying it like this in the project.

Reply:
I'll think about this for some time. The solution should work for all kind of wrapping nodes if possible.